lib/statuses.ex

```diff
@@ -5,113 +5,41 @@ defmodule Statuses do
-        text: :active
-      },
-      %Status{
-        code: 4,
-        desc: "Items marked as done are complete", 
-        text: :done
-      },
-      %Status{
-        code: 5,
-        desc: "A flagged App, Item or Person requires admin attention",
-        text: :flagged
-      },
-      %Status{
-        code: 6,
-        desc:
-          "Soft-deleted items that no longer appear in UI but are kept for audit trail purposes",
-        text: :deleted
-      },
-      %Status{
-        code: 7,
-        desc: "When an email or item is ready to be started/sent is still pending",
-        text: :pending
-      },
-      %Status{
-        code: 8,
-        desc: "An email that has been sent but not yet opened",
-        text: :sent
-      },
-      %Status{
-        code: 9, 
-        desc: "When an email is opened by the recipient", 
-        text: :opened
-      },
-      %Status{
-        code: 10,
-        desc: "Temporary email bounce e.g. because inbox is full",
-        text: :bounce_transient
-      },
-      %Status{
-        code: 11,
-        desc: "Permanent email bounce e.g. when inbox doesn't exist",
-        text: :bounce_permanent
-      },
-      %Status{
-        code: 200, 
-        desc: "successful HTTP request", 
-        text: :ok
-      },
-      %Status{
-        code: 307,
-        desc: "the request should be repeated with another URI",
-        text: :temporary_redirect
-      },
-      %Status{
-        code: 308,
-        desc: "all future requests should be directed to the given URI",
-        text: :permanent_redirect
-      },
-      %Status{
-        code: 400,
-        desc: "server cannot or will not process the request due to an apparent client error",
-        text: :bad_request
-      },
-      %Status{
-        code: 401,
-        desc: "when authentication is required and has failed",
-        text: :unauthorized
-      },
-      %Status{
-        code: 403, 
-        desc: "request forbidden", 
-        text: :forbidden
-      },
-      %Status{
-        code: 404, 
-        desc: "requested resource could not be found",
-        text: :not_found
-      },
-      %Status{
-        code: 429,
-        desc: "has sent too many requests in a given amount of time",
-        text: :too_many_requests
-      },
-      %Status{
-        code: 500,
-        desc: "an unexpected condition was encountered",
-        text: :internal_server_error
-      }
-    ]
+  def get_statuses do
+    parse_json()
+    |> Enum.map(fn obj ->
+      struct(Status, code: obj["code"], desc: obj["desc"], text: String.to_atom(obj["text"]))
+    end)
+  end
+
+  defp parse_json do
+    {:ok, cwd} = File.cwd()
+
+    # we need this `cd` to locate the file in `/deps`
+    case cwd =~ "/statuses" do
+      true ->
+        File.read!("statuses.json")
+        |> Jason.decode!()
+
+      # coveralls-ignore-start
+
+      # temporarily `cd` into `deps/quotes` dir and read `quotes.json` file:
```

Comment:
This appears to have been copy-pasted from `quotes`. 
Consider updating to be correct with the `statuses` project. 💭

Reply:
Sorry about that. Should be fixed in [`5afc679` (#21)](https://github.com/dwyl/statuses/pull/21/commits/5afc679899c0e80f10eb46067230370e0c4bd480).